src/char/int_storage.c

```diff
@@ -249,23 +260,44 @@ static bool inter_storage_guild_storage_tosql(int guild_id, const struct guild_s
 	}
 	SQL->FreeResult(inter->sql_handle);
 
-	chr->memitemdata_to_sql(p->items, guild_id, TABLE_GUILD_STORAGE);
-	ShowInfo ("guild storage save to DB - guild: %d\n", guild_id);
+	int err_count = chr->memitemdata_to_sql(gstor->items.data, gstor->guild_id, TABLE_GUILD_STORAGE);
```

Comment:
An empty storage's `gstor->items.data` is set to `NULL` in `inter_storage_guild_storage_fromsql()` and `mapif_parse_SaveGuildStorage()`, but `char_memitemdata_to_sql()` expects it non-null